### PR TITLE
[Azure Search] Introduce the minimumPrecision parameter for Entity recognition skill

### PR DESCRIPTION
**Pull request by arv100kri**
`minimumPrecision` was already exposed by the REST API, but it was reserved.
Soon (next couple of weeks), that parameter will be used to filter out low confidence entities.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by arv100kri** [comment]
FYI: @brjohnstmsft

File: specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceListSkillsets.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
                 "categories": [
                   "organization"
                 ],
-                "defaultLanguageCode": "en"
+                "defaultLanguageCode": "en",
+					      "minimumPrecision": 0.7
```
Comment (brjohnstmsft) [inline specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceListSkillsets.json]: nit: Weird indentation

**Review by bitsforthought — APPROVED** [review]

**Review by sarangan12 — APPROVED** [review]

**Comment by sarangan12** [comment]
Adding @shahabhijeet & @ds-ms to get their opinion on .NET SDK generation failure

**Comment by dsgouda** [comment]
@sarangan12 Per RPs request we are skipping code generation here as posted in the error messages. We can turn those into Warnings if you'd prefer.
tldr; expected error, disregard
